setup/tue.bash_functions

```diff
@@ -719,7 +729,7 @@ function tue-robocup-update
     cd ~/.tue
     git pull --ff-only
 
-    tue-robocup-install-package picaso_4d_systems
+    tue-robocup-install-package picaso_4d_systems #WHY????
```

Comment:
Indeed, why? 
This line is added in https://github.com/tue-robotics/tue-env/commit/2fa40d7a3b818fac127963a5ce801462495495aa by @svddries and I think it was for a touch screen we had back then. Relatively sure this is OK for removal again.